.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "parquet-hadoop/submodules/parquet-testing"]
```

Comment:
Yes, master is the default. 
There is another file added (here it is parquet-hadoop/submodules/parquet-testing) that points to the exact commit. It will be updated to the latest commit only if we run "git submodule update --remote". 

The problem with moving the submodule to parquet-mr is that we'll have to refer to it from within the code using a relative path "../", which seems pretty problematic to me.
However, I will do a commit moving the submodule up and will see whether Travis is OK with it.

Reply:
Thanks for trying out the move. Hope, it'll work.

The main issue with the git submodule is it does not seem to work automatically. My concern is if this change goes in we will have a lot of issues from developers who cannot build parquet. We need to do something that ensures no regression will occur with our manual (and automatic) build process.

Reply:
I'm adding exec-maven-plugin to address that.